eclair-core/src/test/scala/fr/acinq/eclair/io/PeerSpec.scala

```diff
@@ -218,7 +239,7 @@ class PeerSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with Paralle
     val probe = TestProbe()
     switchboard.send(peer, Peer.Init(Set.empty))
 
-    awaitCond {
+    eventually {
       probe.send(peer, Peer.GetPeerInfo(None))
       probe.expectMsgType[Peer.PeerInfo].state == Peer.DISCONNECTED
```

Comment:
```suggestion
      assert(probe.expectMsgType[Peer.PeerInfo].state == Peer.DISCONNECTED)
```

Reply:
Right! Went too fast on that one and assumed the `probe.expectMsgType[Peer.PeerInfo]` would throw